MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/TurfLineDistanceActivity.java

```diff
@@ -0,0 +1,215 @@
+package com.mapbox.mapboxandroiddemo.examples.javaservices;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.mapbox.geojson.Feature;
+import com.mapbox.geojson.LineString;
+import com.mapbox.geojson.Point;
+import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.maps.Style;
+import com.mapbox.mapboxsdk.style.layers.CircleLayer;
+import com.mapbox.mapboxsdk.style.layers.LineLayer;
+import com.mapbox.mapboxsdk.style.sources.GeoJsonSource;
+import com.mapbox.turf.TurfMeasurement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import static com.mapbox.mapboxsdk.style.layers.Property.LINE_JOIN_ROUND;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.circleColor;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.circleRadius;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineColor;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineJoin;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineWidth;
+import static com.mapbox.turf.TurfConstants.UNIT_KILOMETERS;
+
+/**
+ * Use {@link com.mapbox.turf.TurfMeasurement#distance(Point, Point)} to measure the distance of a drawn line.
+ */
+public class TurfLineDistanceActivity extends AppCompatActivity implements MapboxMap.OnMapClickListener {
+
+  private static final String SOURCE_ID = "SOURCE_ID";
+  private static final String CIRCLE_LAYER_ID = "CIRCLE_LAYER_ID";
+  private static final String LINE_LAYER_ID = "LINE_LAYER_ID";
+
+  // Adjust private static final variables below to change example's UI
+  private static final int CIRCLE_COLOR = Color.RED;
+  private static final int LINE_COLOR = CIRCLE_COLOR;
+  private static final float CIRCLE_RADIUS = 6f;
+  private static final float LINE_WIDTH = 4f;
+  private static String DISTANCE_UNITS = UNIT_KILOMETERS; // Must be a String in {@link com.mapbox.turf.TurfConstants}
```

Comment:
This could be a `private static String`.

Reply:
fixed